app/models/comment.rb

```diff
@@ -91,6 +91,18 @@ def parent
     aid.zero? ? node : answer&.node
   end
 
+  def status_value
+    if status == 0
+      'Banned'
+    elsif status == 1
+      'Normal'
+    elsif status == 4
+      'Moderated'
+    else
+      'Not Defined'  
+    end
+  end
+  
```

Comment:
Looks great, could you please remove this line and remove the space after 'Not defined' so that codeclimate check can pass? We should be ready to merge after that. Thanks

Reply:
@cesswairimu @SidharthBansal Why some tests are not successful?
Do i need to change anything else?
Thanks